test/unit/test_disk_tensor.py

```diff
@@ -13,7 +13,7 @@ def compare_weights_both(url):
   torch_weights = get_state_dict(torch.load(fn, map_location=torch.device('cpu')), tensor_type=torch.Tensor)
   assert list(tg_weights.keys()) == list(torch_weights.keys())
   for k in tg_weights:
-    if tg_weights[k].dtype == dtypes.bfloat16: tg_weights[k] = torch_weights[k].float() # numpy doesn't support bfloat16
+    if tg_weights[k].dtype == dtypes.bfloat16: tg_weights[k] = tg_weights[k].float() # numpy doesn't support bfloat16
```

Comment:
Huh?!?

Reply:
broke after this https://github.com/tinygrad/tinygrad/pull/2698/files#diff-7884618562083b1467b1b4120d56fb3c2d5d2945d328afe4f170365932a60dedR16